### PR TITLE
feat(config): add Inovelli LZW31 Black Series Dimmer

### PR DESCRIPTION
**Pull request by varet80**
Adding a Black Series Dimmer, similar to Red but with different product type

### CODE REVIEW

**Comment by AlCalzone** [comment]
re-opening to test against the newly-added checks

File: packages/config/config/devices/0x031e/lzw31-bsd.json [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+            "maxValue": 101,
+            "defaultValue": 101,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "5": {
+            "label": "Minimum Dim Level",
+            "description": "Minimum Dim Level",
+            "unit": "percent",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw31-bsd.json]:
```suggestion
            "unit": "%",
```

File: packages/config/config/devices/0x031e/lzw31-bsd.json [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+            "maxValue": 45,
+            "defaultValue": 1,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "6": {
+            "label": "Maximum Dim Level",
+            "description": "Maximum Dim Level",
+            "unit": "percent",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw31-bsd.json]:
```suggestion
            "unit": "%",
```

File: packages/config/config/devices/0x031e/lzw31-bsd.json [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+            "maxValue": 32767,
+            "defaultValue": 0,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "9": {
+            "label": "Default Level (Local)",
+            "description": "Default Level (Local)",
+            "unit": "percent",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw31-bsd.json]:
```suggestion
            "unit": "%",
```

File: packages/config/config/devices/0x031e/lzw31-bsd.json [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+            "maxValue": 99,
+            "defaultValue": 0,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "10": {
+            "label": "Default Level (Z-Wave)",
+            "description": "Default Level (Z-Wave)",
+            "unit": "percent",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw31-bsd.json]:
```suggestion
            "unit": "%",
```

File: packages/config/config/devices/0x031e/lzw31-bsd.json [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+            "maxValue": 10,
+            "defaultValue": 3,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "18": {
+            "label": "Active Power Reports",
+            "description": "Active Power Reports",
+            "unit": "percent",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw31-bsd.json]:
```suggestion
            "unit": "%",
```

File: packages/config/config/devices/0x031e/lzw31-bsd.json [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+            "maxValue": 32767,
+            "defaultValue": 3600,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "20": {
+            "label": "Energy Reports",
+            "description": "Energy Reports",
+            "unit": "percent",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw31-bsd.json]:
```suggestion
            "unit": "%",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
just some nits

**Comment by varet80** [comment]
@AlCalzone i pushed a change with this correction but also corrected the source file (used for this device)

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**